examples/docs_beta_snippets/docs_beta_snippets/guides/data-modeling/partitioning/two_dimensional_partitioning.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+import pandas as pd
+
+import dagster as dg
+
+# Create two PartitionDefinitions
+daily_partitions = dg.DailyPartitionsDefinition(start_date="2024-01-01")
+region_partitions = dg.StaticPartitionsDefinition(["us", "eu", "jp"])
+two_dimensional_partitions = dg.MultiPartitionsDefinition(
+    {"date": daily_partitions, "region": region_partitions}
+)
+
+
+# Define the partitioned asset
+@dg.asset(partitions_def=two_dimensional_partitions)
+def daily_regional_sales_data(context: dg.AssetExecutionContext) -> None:
+    multi_partition_key: dg.MultiPartitionKey = context.partition_key.keys_by_dimension
```

Comment:
it might be nice to show how someone could parse this thing into the date key and the region key

Reply:
you got me here - was just being lazy but alright will do 🫡